sprout/appliances/models.py

```diff
@@ -265,6 +265,20 @@ class Group(MetadataMixin):
     def templates(self):
         return Template.objects.filter(template_group=self).order_by("-date", "provider__id")
 
+    @property
+    def appliances(self):
+        return Appliance.objects.filter(template__template_group=self)
+
+    def get_fulfillment_state(self, preconfigured):
```

Comment:
Am I right in thinking that the result is a percentage, if so, I think the work state implies a set of static states, FULL, NOT FULL, ERROR something like that.